### PR TITLE
feat: add Forgot Password API and UI Page

### PR DESCRIPTION
**Pull request by tw-sithumyo**

### CODE REVIEW

File: packages/acquirer-backend/src/routes/userControllers/forgotPassword.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import { type AuthRequest } from '../../types/express'
+import { PortalUserEntity } from '../../entity/PortalUserEntity'
+import { readEnv } from '../../setup/readEnv'
+import { sendForgotPasswordEmail } from '../../utils/sendGrid'
+import { JwtTokenEntity } from '../../entity/JwtTokenEntity'
+import { EmailVerificationTokenEntity } from '../../entity/EmailVerificationToken'
+
+const JWT_SECRET = readEnv('JWT_SECRET', 'secret') as string
+
+/**
```
Comment (elnyry-sam-k) [inline packages/acquirer-backend/src/routes/userControllers/forgotPassword.ts:17]: Is this commented section still required ?

Comment (tw-sithumyo) [reply packages/acquirer-backend/src/routes/userControllers/forgotPassword.ts:17]: yes it is still required... openapi documentation will scan through `routes` directory and generate the open api file...
if i remove the comments, controller doc will also be removed automatically.

This is where generation happens.
https://github.com/mojaloop/merchant-registry-svc/blob/main/packages/acquirer-backend/src/setup/openapiSpecConfig.ts

Comment (elnyry-sam-k) [reply packages/acquirer-backend/src/routes/userControllers/forgotPassword.ts:17]: If it is required, why is it commented?

Comment (tw-sithumyo) [reply packages/acquirer-backend/src/routes/userControllers/forgotPassword.ts:17]: hi @elnyry-sam-k  they are special annotations that are used by the OpenAPI documentation generator to create an accurate and up-to-date API specification document, It is needed.

**Review by elnyry-sam-k — DISMISSED** [review]

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**